test/test.js

```diff
@@ -27,6 +27,23 @@ var allTests = [
 
 p5.prototype.masterVolume(0);
 
+var test_has_run = false;
+
 require(allTests, function(){
-  mocha.run();
+  document.getElementById('mocha').innerHTML = 'click to begin tests';
+
+  // chromes autoplay policy requires a user interaction
+  // before the audiocontext can activate
+  function mousePressed() {
+    if(!test_has_run) {
+      document.getElementById('mocha').innerHTML = '';
+      getAudioContext().resume()
```

Comment:
Can we use https://p5js.org/reference/#/p5/userStartAudio (in this case, `p5.prototype.userStartAudio`) for this?

Reply:
This works! Just pushed new commit. When requiring p5, what is the best way to include the p5.js library? I was running into some issues before realizing that libs/p5.js was missing, which was resolved after manually importing the file, but I don't know if I saw anything about that in the contributions docs.